.github/workflows/ci.yml

```
@@ -62,24 +62,25 @@ jobs:
       - run: npm ci
       - run: npx --no-install eslint .
 
-  docker-buildx:
-    name: Docker Buildx
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        platform: [ linux/amd64, linux/arm64 ]
-        app: [ legacy-api, ocean-api ]
-    steps:
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@27d0a4f181a40b142cce983c5393082c365d1480
+  # TODO(kodemon): Re-enable in subsequent docker PRs
+  # docker-buildx:
+  #   name: Docker Buildx
+  #   runs-on: ubuntu-latest
+  #   strategy:
+  #     matrix:
+  #       platform: [ linux/amd64, linux/arm64 ]
```

```suggestion
       platform: [ linux/amd64 ]
```

We should just remove `arm64` build since that's the slow one.

---

That's fair, removed `arm64` 👍🏻